e2e_tests/src/e2e_indexer/e2elive.py

```diff
@@ -316,5 +345,38 @@ def start(self):
         self.t.start()
 
 
+def trace(func, lg):
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        lg.debug(f"<<<<<Entering {func.__name__}>>>>>")
+        result = func(*args, **kwargs)
+        lg.debug(f"<<<<<Exiting {func.__name__}>>>>>")
+        return result
+
+    return wrapper
+
+
+def wrap_funcs(pymod):
+    is_va = pymod is e2e_indexer.validate_accounting
+    for name, obj in inspect.getmembers(pymod):
```

Comment:
clever but kind of nutty?

Reply:
Just to produce breadcrumbs without having to modify every function directly. Any objections?